api/post_test.go

```diff
@@ -630,19 +628,19 @@ func TestSearchPostsFromUser(t *testing.T) {
 	// wait for the join/leave messages to be created for user3 since they're done asynchronously
 	time.Sleep(100 * time.Millisecond)
```

Comment:
We do want to wait for the join/leave messages because we want to make sure the search doesn't return them